source/usage-examples.txt

```diff
@@ -4,4 +4,51 @@ Usage Examples
 
 .. default-domain:: mongodb
 
-Lorem ipsum
+Overview
+--------
+
+Usage examples provide convenient starting points for popular MongoDB
+operations. Each example provides:
+
+- an explanation of the operation in the example showing the
+  purpose and a sample use case for the method
+
+- an explanation of how to use the operation, including parameters,
+  return values, and common exceptions you might encounter
+
+- a full Java class that you can copy and paste to run the example
+  in your own environment
+
+How to Use the Usage Examples
+-----------------------------
+
+These examples use the :atlas:`sample datasets <sample-data>` provided by
+Atlas. You can load them into your database on the free tier of MongoDB
+Atlas by following the
+:atlas:`Get Started with Atlas Guide <getting-started/#atlas-getting-started>`
+or you can
+:guides:`import the sample dataset into a local MongoDB instance
+</server/import/>`.
+
+Once you have imported the dataset, you can copy and paste a usage
+example into your development environment of choice. You can follow the
+:doc:`quick start guide </quick-start>` to learn more about getting
+started with the MongoDB Java driver. Once you've copied a usage example,
+you'll need to edit the connection string to get the example connected to
+your instance of MongoDB:
+
+.. code-block:: java
+
+   // Replace the following with your MongoDB deployment's connection string.
+   String uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
```

Comment:
I mention this in another PR as well: should this variable be named `connectionString` to avoid introducing additional terminology (technically our connection strings are not valid URIs according to the RFC)?

Reply:
I'm happy to change it, but am not sure what to do about all the other instances in which we use URI across our docs. Just a few examples:

"Connection String URI Format" https://docs.mongodb.com/manual/reference/connection-string/

"URI string"
https://docs.mongodb.com/guides/server/drivers/

"Connection URI"
https://docs-mongodbcom-staging.corp.mongodb.com/17459db/node/docsworker-xlarge/preview-040120/fundamentals/connection

Reply:
I figured as much.  I'll resolve.